docs/content/integrations/embedded-elt/dlt.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Dagster Embedded ELT: dlt"
+description: Ingest data with ease using Dagster and dlt
+---
+
+## dlt
+
+To get started with `dagster-embedded-elt` and _dlt_, first familiarize yourself with the quick [introduction to dlt](https://dlthub.com/docs/intro). When using the _dlt_ integration, you can either write your own sources, or use one of the many pre-made [verified sources](https://dlthub.com/docs/dlt-ecosystem/verified-sources/) in the _dlt_ ecosystem.
+
+## 1. Configuring your Dagster project to support `dlt`
```

Comment:
I think this jumps too quickly into 'tutorial' mode. For people who are not wanting to do a tutorial, we should do a better job explaining the high level concepts they should be familiar with.

Reply:
Thank you - I added a background section, but it may need more. We'll see!